lib/classes/hint.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:math';
+
+import 'package:phone_factory/classes/Ship.dart';
+import 'package:phone_factory/classes/bay_item.dart';
+
+class Hint {
+  int _currentBayNumber;
+  Ship ship = Ship();
```

Comment:
create Ship only one time in Game class. and path created data via function arguments or class constructors